source/ranges.tex

```diff
@@ -5120,29 +5120,26 @@
     constexpr auto begin() const
       requires @\libconcept{random_access_range}@<const V> && @\libconcept{sized_range}@<const V>;
 
-    constexpr auto end()
-      requires (!@\exposconcept{simple-view}@<V>)
+    constexpr auto end() requires (!@\exposconcept{simple-view}@<V>)
     { return ranges::end(@\exposid{base_}@); }
 
-    constexpr auto end() const
-      requires @\libconcept{range}@<const V>
+    constexpr auto end() const requires @\libconcept{range}@<const V>
     { return ranges::end(@\exposid{base_}@); }
 
-    constexpr auto size()
-      requires @\libconcept{sized_range}@<V>
+    constexpr auto size() requires @\libconcept{sized_range}@<V>
     {
       const auto s = ranges::size(@\exposid{base_}@);
       const auto c = static_cast<decltype(s)>(@\exposid{count_}@);
       return s < c ? 0 : s - c;
     }
 
-    constexpr auto size() const
-      requires @\libconcept{sized_range}@<const V>
+    constexpr auto size() const requires @\libconcept{sized_range}@<const V>
     {
```

Comment:
This brace should be on the previous line.

Reply:
Fixed.